subsys/bluetooth/host/hci_core.h

```diff
@@ -504,6 +504,9 @@ void bt_finalize_init(void);
 
 void bt_hci_host_num_completed_packets(struct net_buf *buf);
 
+/* Get bondable flag */
+bool bt_get_bondable(void);
+
```

Comment:
I would like this to be a public function, like `bt_set_bondable` is.

Reply:
Currently, I haven’t found any need for this outside of stack, so I made it an internal function.
Public it is OK for me.

Reply:
Agree with @alwa-nordic - Since setting `bondable` is a public function, getting it might as well be that too.